.github/workflows/dependabot_reviewer.yml

```diff
@@ -23,6 +23,24 @@ jobs:
         uses: dependabot/fetch-metadata@v1.5.1
         with:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
+      # For the current supported version, update dependencies according to docs
+      - name: Get list of supported version and backport dependencies
+        id: supported-versions
+        run: |
+          supportedVersions=$(jq -r '.versions[] | select(.support_end_date > (now | strftime("%Y-%m-%d"))) | .branch | "backport \(.)"' supported_versions.json | paste -sd, -)
+          
+          # Join array elements with commas
+          joinedVersions=$(printf "%s, " "${supportedVersions[@]}")
+          joinedVersions=${joinedVersions%, }  # Remove the trailing comma     
```

Comment:
I think we can use `jq`'s `join()` function to simplify this: https://jqlang.github.io/jq/manual/#:~:text=Example-,join(str),-Joins%20the%20array